server/src/test/java/org/elasticsearch/search/SearchServiceTests.java

```diff
@@ -178,6 +178,12 @@ public void onIndexModule(IndexModule indexModule) {
         }
     }
 
+    @Override
+    protected boolean enableConcurrentSearch() {
+        // we don't test search concurrency specifically but we want to verify default behaviour hence we disable it.
+        return false;
+    }
+
```

Comment:
I think we do want to test search concurrency specifically if that is going to be the path forward. 

There are tests that break here when this is `false`, and they would otherwise need to be moved to their own duplicate class where this is enabled.

Reply:
I removed this and updated the tests in this class to expect concurrency by default.